File: MyBudgetSpring/src/main/java/com/example/mybudgetspring/model/dto/Account.java
package com.example.mybudgetspring.model.dto;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Account implements Serializable {
    private Integer accountId;
    private String name;
    private String currency;
    private Double balance;
}

File: MyBudgetAngular/mybudgetapp/src/app/model/account.ts
export class Account {
  accountId: number;
  name: string;
  currency: string;
  balance: number;
  defaultCurrencyBalance: number;

  constructor(
    accountId: number,
    name: string,
    currency: string,
    balance: number,
    defaultCurrencyBalance: number
  ) {
    this.accountId = accountId;
    this.name = name;
    this.currency = currency;
    this.balance = balance;
    this.defaultCurrencyBalance = defaultCurrencyBalance;
  }
}
